src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reversed = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (reversed[value] !== undefined) {
+      return null;
```

Comment:
It works, but you can also try
https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Operators/in
for a cleaner solution

Reply:
Oh, thx. This is really useful)